forge/routes/api/admin.js

```diff
@@ -4,13 +4,18 @@ module.exports = async function (app) {
     app.get('/stats', async (request, reply) => {
         const userCount = await app.db.models.User.count({ attributes: ['admin'], group: 'admin' })
         const projectStateCounts = await app.db.models.Project.count({ attributes: ['state'], group: 'state' })
+        const license = await app.license.get() || {}
```

Comment:
I feel we should find a way to populate this with the default values if no license present, then we don't have hard coded values in the UI portion that we may need to change later.

Reply:
```suggestion
        const license = await app.license.get() || app.license.defaults
```

Reply:
Sure i will change it. Thanks for the suggestion.